_docs/geyser/port-forwarding.md

```diff
@@ -134,12 +134,36 @@ SoYouStart (subsidiary of OVH):
 5. Save and wait a few seconds for the changes to come into effect.
 
 ### Oracle Cloud/OCI
-You'll need to allow port 19132 on UDP in both the security list and the firewall-cmd command.
+By default, Oracle Cloud will block all incoming traffic except for SSH and RDP. This must be resolved within Oracle Cloud itself and the Compute Instance running Geyser.
+
+The steps below assume that you are using the default ports for the Java server and Geyser, and should be adjusted accordingly.
+
+1. Find your Compute Instance in the OCI Console
+2. Click on the instances Virtual Cloud Network (this will be under "Instance details")
+3. On the left-hand side, select "Security Lists"
+4. Select one of the security lists. By default only one security list will exist. It doesn't matter which security list we add the rules to.
+5. Select "Add Ingress Rules"
+6. Set "Source CIDR" to `0.0.0.0/0`
+7. Set "Destination Port Range" to `25565-25565`
+8. Select "Another Ingress Rule"
+9. Repeat steps 6 and 7, but with a "Destination Port Range" of `19132-19132`
```

Comment:
Can we mention that the first rule would be for the Java server (and therefore optional for geyser standalone), and the second for Geyser?

Reply:
Clarified that this is for Java and is optional